logging/README.md

```diff
@@ -257,6 +257,48 @@ The script creates the namespace into which the components are deployed. By defa
 Updates in place are supported. To update, re-run the
 `deploy_logging_open.sh` script to install the latest versions of all components, indexes, and dashboards.
 
+## Important Information about Kibana in the New Release
+
```

Comment:
I think we need a better section title.  This document is the top-level README and will be here forever (more or less).  Given that, the phrase "new release" isn't meaningful.  Maybe something like:  "Using Kibana"?

Reply:
The title is meaningful at this time, catches attention in a sea of instructions, and gives weight to make sure users do not ignore the information. We can adjust the title as time goes by. Indeed, the topic itself will likely change significantly as time goes by.